### ISSUE TITLE
Report violation if required a11y metadata 'conformsTo' is absent.

### ISSUE THREAD

**Issue by marisademeglio** [daisy/ace#171]
See
http://www.idpf.org/epub/a11y/accessibility.html#sec-conf-reporting

**Comment by laudrain** [comment]
We had discussions about that in France in "Normes&Standards" working group.
We are thoughtful about self-certification by the publisher, and considering a 2 steps process:
- publishers (and epub producers) setup a conformance level
- independant organisations can verify the conformance and declare certification
Then the certifiedBy property would not be setup at production level.
Are we diverging from this specification?

**Comment by rdeltour** [comment]
> Are we diverging from this specification?

No, the 2-step approach you’re describing sounds OK!

For clarification: the absence of `conformsTo` would **not** be reported as a violation, but the (its value or its absence) property should be reported along with other properties in the "Accessibility metadata" section.

**Comment by laudrain** [comment]
@rdeltour: thanks for the clarification.

**Comment by mattgarrish** [comment]
> should be reported along with other properties in the "Accessibility metadata" section

It's currently only reported in the links section when present.

**Comment by marisademeglio** [comment]
@rdeltour Is there any other property that would be in `links` that we want in the metadata table? I'm just running into an issue where this link property gets special treatment, separate from the others. It's fine but ugly.

Also, if we do want to include other `link` properties, should they just have their own table? Or should we rename this one "metadata and links" ?

**Comment by rdeltour** [comment]
> @rdeltour Is there any other property that would be in links that we want in the metadata table? 

Looking at the [link vocabulary](https://idpf.github.io/epub-vocabs/package/link/), I don’t think so.

Well, we could of course try to fetch and parse external metadata records, but I don’t think we want to go there just yet 🙂

> Also, if we do want to include other link properties, should they just have their own table? Or should we rename our current one "metadata and links" ?

The `conformsTo` info should really be at the same place as the other a11y metadata info, so I wouldn’t put it in its own table. Renaming the table is fine, but then it’s a long name, does it integrate well in the UI?

**Comment by marisademeglio** [comment]
> The conformsTo info should really be at the same place as the other a11y metadata info, so I wouldn’t put it in its own table. Renaming the table is fine, but then it’s a long name, does it integrate well in the UI?

It's not as nice. We will just lump this one link property in with the metadata and call it all "metadata".

**Comment by jenstroeger** [comment]
Helpful discussion, thank you!

Is it possible to _derive_ the value of `conformsTo` from the results of an Ace run? Or need I provide that [conformance level value](http://www.idpf.org/epub/a11y/accessibility.html#sec-conf-reporting) myself? And if so, will that value be validated against the state of the document?
